plugins/unicodelib/unicodelib.cpp

```diff
@@ -1738,3 +1779,8 @@ UNICODELIB_API void UNICODELIB_CALL ulUnicodeLocaleRemoveSuffix(unsigned & tgtLe
     tgt = (UChar *)CTXMALLOC(parentCtx, tgtLen * 2);
     pro.extract(0, tgtLen, tgt);
 }
+
+UNICODELIB_API void UNICODELIB_CALL ulUnicodeLocaleRepeat(unsigned & tgtLen, UChar * & tgt, unsigned srcLen, UChar const * src, unsigned n)
+{
+    repeat(tgtLen, tgt, srcLen, src, n);
```

Comment:
Why is repeat a separate function rather than just being inline here?

Reply:
because that is the structure David had used when he started on this branch, and I didn't change it.  I will inline it.